### PR TITLE
Fill in feature issue number in feature_set

### PR DESCRIPTION
**Pull request by CriesofCarrots**
#### Problem
Feature issue number of `prevent_rent_paying_rent_recipients` is mysterious

#### Summary of Changes
Fill in correct number

### CODE REVIEW

File: sdk/src/feature_set.rs [hunk 1]
```diff
@@ -756,7 +756,7 @@ lazy_static! {
         (cap_transaction_accounts_data_size::id(), "cap transaction accounts data size up to a limit #27839"),
         (remove_congestion_multiplier_from_fee_calculation::id(), "Remove congestion multiplier from transaction fee calculation #29881"),
         (enable_request_heap_frame_ix::id(), "Enable transaction to request heap frame using compute budget instruction #30076"),
-        (prevent_rent_paying_rent_recipients::id(), "prevent recipients of rent rewards from ending in rent-paying state #30???"),
+        (prevent_rent_paying_rent_recipients::id(), "prevent recipients of rent rewards from ending in rent-paying state #30130"),
```
Comment (jbiseda) [inline sdk/src/feature_set.rs:759]: Don't you want #30151?

Comment (CriesofCarrots) [reply sdk/src/feature_set.rs]: Oh, good point, yes! I forgot we switched to linking to the feature-gate issues.

Comment (CriesofCarrots) [reply sdk/src/feature_set.rs]: Updated

**Review by jbiseda — APPROVED** [review]
LGTM

**Pull request merged**